\

```diff
@@ -0,0 +1,19 @@
+module SettingsEmailHelper
```

Comment:
needs a file name

Reply:
shouldn't be in there at all! Must have accidentally added all when I didn't mean to.  I'll get it  gone!